h2o-core/src/main/java/water/ExternalFrameWriterBackend.java

```diff
@@ -109,8 +109,7 @@ private static void storeVector(AutoBuffer ab, NewChunk[] nchnk, int maxVecSize,
         }
 
         // fill remaining zeros
-        int lastIdx = indices.length == 0 ? 0 : indices[indices.length - 1];
```

Comment:
This was problematic part. We were overwriting last value, which also caused that the resulting chunk was at some cases 1 element shorter